app/views/organizations/home/index.html.erb

```diff
@@ -144,14 +144,19 @@
     <div class="row pb-3">
         <div class="col-lg-4 col-md-6 mb-4 mb-lg-0">
             <span>
-                <%= image_tag('danielle_2.jpg', 
+                <%= image_tag('danielle_2.jpg',
                     class: 'rounded-5 w-100 px-4 px-lg-0') %>
             </span>
         </div>
         <div class="col-lg-4 px-4 order-last order-lg-0">
             <p class='justify bigger'>
               <%= Current.organization.page_text&.about || "#{Current.organization.name} was founded by an incredible group of ladies in April of 2020. Our initial goal was to have both a rescue and a spay/neuter clinic, however, we quickly realized that it would be more efficient to separate into two organizations."%>
             </p>
```

Comment:
Can we apply a similar approach to the Hero image for the about us images? I.e., if we have an uploaded image, we use that, otherwise, we use the default?

The about us is a bit different because we have two images, but we could check the number of about us images attached, if 0, we use both default images, if 1, we replace one of the default images, if 2, we replace both of the default images. Let me know if you have any questions.